server/xpub-server/index.js

```diff
@@ -36,7 +32,6 @@ logger.info(appMessage('Starting'))
 const cleanupHandler = cleanup.Cleanup(process, logger, stopServer)
 
 module.exports = {
-  backend: () => registerRoutes,
```

Comment:
I think we still need to register the routes with pubsweet for them to be available. It will most likely be looking for packages that export objects with `backend` properties. maybe this could be done within the `component-elife-app/index.js`?